package.json

```diff
@@ -58,7 +58,8 @@
     "@babel/preset-env": "7.14.9",
     "@justeat/browserslist-config-fozzie": "1.2.0",
     "@justeat/eslint-config-fozzie": "5.1.0",
-    "@justeat/fozzie": "10.5.0",
+    "@justeat/fozzie": "10.x",
+    "@justeat/pie-design-tokens": "3.0.0",
```

Comment:
I would just define this in the package that needs it, rather than at the root of the mono-repo. Otherwise, I think it's potentially confusing, as this dependency version comes from fozzie itself.